src/codeintel/lib/codeintel2/database/langlib.py

```diff
@@ -1365,6 +1420,19 @@ def cull_mem(self):
         #    memory. How old is the oldest one? Estimate memory size
         #    used by all loaded indeces?
 
+    def _cull_cached_blobs(self, min_access_seconds, retain_dbsubpath=None):
+        """
+        Clean up old cached blobs and optionally retain a specific subpath
+
+        Assumes that caller has called _acquire_lock
+        """
+        now = time.time()
+        for dbsubpath, atime in self._blob_cache_atime.items():
+            if dbsubpath != retain_dbsubpath and now - atime > min_access_seconds:
```

Comment:
You don't need to change this but this could be `dbsubpath is not retain_dbsubpath`.